package.json

```diff
@@ -41,21 +40,21 @@
     "@storybook/addons": "^3.3.15",
     "@storybook/react": "^3.3.15",
     "babel-core": "^6.26.0",
-    "babel-plugin-react-intl": "^2.4.0",
+    "babel-plugin-react-intl-auto": "^1.1.0",
     "eslint-config-react-tools": "1.x.x",
+    "extract-react-intl-messages": "^0.8.2",
     "prettier": "1.12.1",
-    "serve": "^6.1.0"
+    "serve": "^6.1.0",
+    "storybook-addon-intl": "^2.3.0"
   },
   "scripts": {
     "start": "npm-run-all -p watch-css start-js",
     "build": "npm-run-all build-css build-js && cp -r dist/* _dist/",
-    "build-langs": "babel intl.buildlangs.js | node",
     "start-js": "react-static start",
     "build-js": "react-static build",
-    "build-css":
-      "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
-    "watch-css":
-      "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
+    "build-translations": "extract-messages -l=en,es,vi,ar -o src/js/i18n/locales/ --flat 'src/**/*.js'",
+    "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
+    "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
```

Comment:
https://github.com/cityofaustin/janis/pull/230

Reply:
can do.

Reply:
hey so, I think your editor install is working because you have node_modules locally. I had to install prettier globally for my editor to work. just fyi, would you mind if I updated the instructions?

Reply:
Yeah I'm pretty sure I had to install it globally as well.